wipac_telemetry/tracing_tools/config.py

```diff
@@ -8,14 +8,14 @@
 
 
 class _TypedConfig(TypedDict):
+    OTEL_EXPORTER_OTLP_ENDPOINT: str
     WIPACTEL_EXPORT_STDOUT: bool
-    WIPACTEL_EXPORT_OTLP: bool
     WIPACTEL_LOGGING_LEVEL: str
 
 
 defaults: _TypedConfig = {
+    "OTEL_EXPORTER_OTLP_ENDPOINT": "",
```

Comment:
This is fine as is. But it doesn't look like this will propagate to the actual exporter.

Reply:
I've approved so you don't have to do anything. A comment here saying that adding a default value won't propagate to the exporter, IOW it's just here to appease mypy may be useful